File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.go
func solution(numer1 int, denom1 int, numer2 int, denom2 int) [2]int {
    a := numer1 * denom2 + numer2 * denom1
    b := denom1 * denom2
    var c [2]int
    
    for i := 1; i<=a && i<=b; i++ {
        if a%i == 0 && b%i == 0 {
            c[0] = a / i
            c[1] = b / i
        }
    }

    return c
}

File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.java
class Solution {
    public int[] solution(int numer1, int denom1, int numer2, int denom2) {
        
        
        int denom = denom1 * denom2;
        int numer = numer1 * denom2 + numer2 * denom1;
        
        int a = 1;
        for (int i=1; i<=numer && i<=denom; i++) {
            if (denom%i == 0 && numer%i == 0) {
                a = i;
            }
        }

        int[] answer = {numer/a, denom/a};
        
        return answer;
    }

}